### PR TITLE
Fixes Elixir 1.4.* warnings

### PR DESCRIPTION
**Pull request by Ilyes512**
My first contribution (if excepted) to an Elixir package :)

I am currently using Elixir 1.4.2 and when I used your package it was giving me warnings like these:

```
warning: variable "fetch_currency" does not exist and is being expanded to "fetch_currency()", please use parentheses to remove the ambiguity or change the variable name
  lib/faker_elixir/currency.ex:20
```

I have "fixed" those warnings.

### CODE REVIEW

File: README.md
Comment (GesJeremie) [inline README.md]: Can you revert this change? I think it's better to wait for the official release 1.0.0 ... ?

File: README.md
Comment (GesJeremie) [inline README.md:28]: Nice 👍

File: .travis.yml
Comment (GesJeremie) [inline .travis.yml:6]: Thanks for that!

File: README.md
Comment (GesJeremie) [inline README.md]: Well that's my bad, I did let ```@credit_card_vendors ["Visa", "MasterCard"]``` in the locales but it's not used. This method already exists and it's called ```FakerElixir.Bank.credit_card_type # "MasterCard```

File: lib/faker_elixir/address.ex
Comment (GesJeremie) [inline lib/faker_elixir/address.ex:243]: 👍

File: lib/faker_elixir/address.ex
Comment (GesJeremie) [inline lib/faker_elixir/address.ex:257]: 👍

File: lib/faker_elixir/bank.ex
Comment (GesJeremie) [inline lib/faker_elixir/bank.ex]: Like said ```credit_card_type``` already does the job ;)

File: lib/faker_elixir/bank.ex
Comment (GesJeremie) [inline lib/faker_elixir/bank.ex]: I would like to keep it that way (but I appreciate)

File: lib/faker_elixir/commerce.ex
Comment (GesJeremie) [inline lib/faker_elixir/commerce.ex:37]: 👍

File: lib/faker_elixir/commerce.ex
Comment (GesJeremie) [inline lib/faker_elixir/commerce.ex:45]: 👍

File: lib/faker_elixir/commerce.ex
Comment (GesJeremie) [inline lib/faker_elixir/commerce.ex:48]: 👍

File: lib/faker_elixir/locales/en.ex
Comment (GesJeremie) [inline lib/faker_elixir/locales/en.ex]: That's something I forgot to drop from the codebase, you could erase it :)

File: lib/faker_elixir/locales/en.ex
Comment (GesJeremie) [inline lib/faker_elixir/locales/en.ex]: Same thing here :)

File: lib/faker_elixir/locales/it.ex
Comment (GesJeremie) [inline lib/faker_elixir/locales/it.ex:24]: Nice catch mate!

**Review by GesJeremie — CHANGES_REQUESTED** [review]

**Comment by GesJeremie** [comment]
Hey @Ilyes512, thanks for your contribution, I did a small code review, I will be happy to merge your PR if you tweak some stuff :)

Let me know if you don't agree with something ;)

File: README.md
Comment (Ilyes512) [reply README.md]: I thought this package was actually on version 0.0.0 and so I almost left the page without checking it out (I thought it would be new and so lack lots of features). I will revert this later.

File: lib/faker_elixir/bank.ex
Comment (Ilyes512) [reply lib/faker_elixir/bank.ex]: I will remove this later and remove all other related code.

**Review by GesJeremie — APPROVED** [review]

**Comment by GesJeremie** [comment]
Good to go, I'm gonna release that. Thanks for your work 👍

File: README.md
Comment (GesJeremie) [reply README.md]: At the end of the day, I think you are right. Wanna get a free commit? :p

Comment (Ilyes512) [reply README.md]: Okey, I will create a MR later and add some extra badges on top of the readme with the current version as well.

Comment (GesJeremie) [reply README.md]: Awesome!

**Pull request closed without merging**
